HW_6/Task_2.js

```diff
@@ -0,0 +1,23 @@
+const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+const myPizzas = ['Caprichosa', 'Romana', 'Siciliana', 'Margarita', 'Hawai'];
+let onlyMyPizzas = [];
+// for (let myPizza of myPizzas){
+//    if (!competitorPizzas.includes(myPizza)){
+//     differentPizzas.push(myPizza);
+//    }
+// }
+
+for (let myPizza of myPizzas){
+    let exist = false;
+    for (let element of competitorPizzas) {
+        if(element.toLowerCase() === myPizza.toLowerCase()) {
+            exist = true;
+        }
+    }
+
+    if(!exist) {
+        onlyMyPizzas.push(myPizza);
+    } 
```

Comment:
```suggestion
    let exist = false;
    for (let element of competitorPizzas) {
        if(element.toLowerCase() === myPizza.toLowerCase()) {
             onlyMyPizzas.push(myPizza);
        }
    }
```
Лучше конечно было бы заранее за 1 цикл перевести пиццы конкурента в нижний регистр и потом сравнивать, будет меньше итераций цикла в сумме)